static/js/global-nav.js

```diff
@@ -0,0 +1 @@
+import * as globalNav from "global-nav"; // eslint-disable-line no-unused-vars
```

Comment:
Is `globalNav` called at all? If it's just imported does it run and show on the page?

Reply:
Oh, seems like I forgot to push something here.

Reply:
Have updated it :)